### PR TITLE
OCTO-2214 Bugfix: Capping information fraction

### PR DESCRIPTION
**Pull request by shansfolder**
There is actually a bug in ```group_sequential``` when actual data size is larger than expected. 

This PR fixed the bug and add a unit test for this case. And added some small improvements of doc.

### CODE REVIEW

File: expan/core/early_stopping.py [hunk 1]
```diff
@@ -67,19 +67,13 @@ def group_sequential(x,
...
-    # else:
-    #	fraction = information_fraction
-
     n_x = statx.sample_size(_x)
     n_y = statx.sample_size(_y)
 
     if not estimated_sample_size:
         information_fraction = 1.0
     else:
-        information_fraction = max(1.0, min(n_x, n_y) / estimated_sample_size)
```
Comment (gbordyugov) [inline expan/core/early_stopping.py:82]: Nice catch, thank you! I think it was me who wrote this line back then ;)

**Review by gbordyugov — COMMENTED** [review]
Great job!

**Review by mkolarek — APPROVED** [review]
:+1:

**Review by daryadedik — APPROVED** [review]

**Comment by shansfolder** [comment]
Alt text

**Pull request merged**